doc/releases/changelog-dev.md

```diff
@@ -111,153 +171,67 @@ Users can specify the control wires as well as the values to control the operati
-  Hamiltonian is defined:
+  - entire QNodes:
 
-  ```python
-  hf_state = qml.qchem.hf_state(n_electrons, qubits)
-  dev = qml.device("default.qubit", wires=qubits)
-  @qml.qnode(dev)
-  def circuit():
-      qml.BasisState(hf_state, wires=range(qubits))
-      return qml.expval(H)
-  ```
+    ```python
+    dev = qml.device("default.qubit", wires=["A", "B", "C", "D"])
+    wire_map = {0: "A", 1: "B", 2: "C", 3: "D"}
 
-  Finally, the optimizer is instantiated and then the circuit is created and optimized adaptively:
+    @qml.qnode(dev) 
+    def circuit():
+        qml.RX(0.54, wires=0)
+        qml.PauliX(1)
+        qml.PauliZ(2)
+        qml.RY(1.23, wires=3)
+        return qml.probs(wires=0)
+    ```
 
-  ```python
-  opt = qml.optimize.AdaptiveOptimizer()
-  for i in range(len(operator_pool)):
-      circuit, energy, gradient = opt.step_and_cost(circuit, operator_pool, drain_pool=True)
-      print('Energy:', energy)
-      print(qml.draw(circuit)())
-      print('Largest Gradient:', gradient)
-      print()
-      if gradient < 1e-3:
-          break
-  ```
-  
-   ```pycon
-  Energy: -1.246549938420637
-  0: ─╭BasisState(M0)─╭G²(0.20)─┤ ╭<𝓗>
-  1: ─├BasisState(M0)─├G²(0.20)─┤ ├<𝓗>
-  2: ─├BasisState(M0)─│─────────┤ ├<𝓗>
-  3: ─├BasisState(M0)─│─────────┤ ├<𝓗>
-  4: ─├BasisState(M0)─├G²(0.20)─┤ ├<𝓗>
-  5: ─╰BasisState(M0)─╰G²(0.20)─┤ ╰<𝓗>
-  Largest Gradient: 0.14399872776755085
-
-  Energy: -1.2613740231529604
-  0: ─╭BasisState(M0)─╭G²(0.20)─╭G²(0.19)─┤ ╭<𝓗>
-  1: ─├BasisState(M0)─├G²(0.20)─├G²(0.19)─┤ ├<𝓗>
-  2: ─├BasisState(M0)─│─────────├G²(0.19)─┤ ├<𝓗>
-  3: ─├BasisState(M0)─│─────────╰G²(0.19)─┤ ├<𝓗>
-  4: ─├BasisState(M0)─├G²(0.20)───────────┤ ├<𝓗>
-  5: ─╰BasisState(M0)─╰G²(0.20)───────────┤ ╰<𝓗>
-  Largest Gradient: 0.1349349562423238
-
-  Energy: -1.2743971719780331
-  0: ─╭BasisState(M0)─╭G²(0.20)─╭G²(0.19)──────────┤ ╭<𝓗>
-  1: ─├BasisState(M0)─├G²(0.20)─├G²(0.19)─╭G(0.00)─┤ ├<𝓗>
-  2: ─├BasisState(M0)─│─────────├G²(0.19)─│────────┤ ├<𝓗>
-  3: ─├BasisState(M0)─│─────────╰G²(0.19)─╰G(0.00)─┤ ├<𝓗>
-  4: ─├BasisState(M0)─├G²(0.20)────────────────────┤ ├<𝓗>
-  5: ─╰BasisState(M0)─╰G²(0.20)────────────────────┤ ╰<𝓗>
-  Largest Gradient: 0.00040841755397108586
-  ``` 
+    ```pycon
+    >>> mapped_circuit = qml.map_wires(circuit, wire_map)
+    >>> mapped_circuit()
+    tensor([0.92885434, 0.07114566], requires_grad=True)
+    >>> print(qml.draw(mapped_circuit)())
+    A: ──RX(0.54)─┤  Probs
+    B: ──X────────┤       
+    C: ──Z────────┤       
+    D: ──RY(1.23)─┤   
+    ```
 
 <h3>Improvements</h3>
 
-* Added the `samples_computational_basis` attribute to the `MeasurementProcess` and `QuantumScript` classes to track
-  if computational basis samples are being generated.
+* Added a `samples_computational_basis` attribute to the `MeasurementProcess` and `QuantumScript` classes to track if computational basis samples are being generated.
```

Comment:
Needs more context

Reply:
This is also a more internal change. We allow the syntax `qml.sample()` or `qml.counts()` where we don't specify  what observable we are sampling. We assume it to be the computational basis. This attribute (accessible by doing something like `mp = qml.sample()  #mp.samples_computational_basis == True`) just tracks that special case. 

We needed it to do some validation checks and make sure we are not measuring things which can't be measured together!